### DISCUSSION TITLE
2024-06-01(토)

### DISCUSSION THREAD

**Discussion by Seoin02** [Together-3team/petFrontend#76] [데일리스크럼]
아래 내용을 복사해서 작성해주시면 됩니다.
```
### 완료한 일

### 해야할 일

### 어려운 일
```

**Comment by devwqc** [comment]
### 완료한 일
- 루트 레이아웃 padding 작업
- 배너 캐러셀 컴포넌트 작업

### 해야할 일
- 배너 캐러셀 컴포넌트 구현
- 카테고리 버튼 컴포넌트 구현
- 스크롤 Top 버튼 컴포넌트 구현

### 어려운 일
- 아자아자!💪

**Comment by q45402sk** [comment]
### 완료한 일

### 해야할 일
zzim 컴포넌트 기능 구현

### 어려운 일

**Comment by minyoung0503** [comment]
### 완료한 일
- Textarea css 수정
- 대표 리뷰 컴포넌트

### 해야할 일
- 리뷰 전체보기 페이지
- 리뷰 정렬 dropdown

### 어려운 일
- 리뷰 정렬 dropdown

**Comment by Seoin02** [comment]
### 완료한 일

### 해야할 일
- signup 페이지 코드리뷰 반영
- 커스텀 인풋 공통 컴포넌트 구현
- onboarding 페이지 UI 구현

### 어려운 일
- 생각보다 페이지 구현 작업 시간이 더 걸린다...
